desktop/components/split_test/running_tests.coffee

```diff
@@ -14,4 +14,9 @@
 # this should export empty Object
 # module.exports = {}
 
-module.exports = {}
+module.exports =
```

Comment:
@mzikherman let me reiterate to make sure I understand correctly. You're suggesting I pull out the wiring up of the a/b test until this is signed off as good to go in the QA phase and replacing it with a query parameter that triggers the new header search styles for the time being?